dist/stampit.js

```diff
@@ -710,9 +738,13 @@ isStamp = function isStamp(obj) {
  * @return {Function}             A composable stampit factory
  *                                (aka stamp).
  */
-convertConstructor = function convertConstructor(Constructor) {
-  return stampit().methods(Constructor.prototype).enclose(Constructor);
-};
+function convertConstructor(Constructor) {
+  var stamp = stampit();
+  mixInChainFunctions(stamp.fixed.methods, Constructor.prototype);
+  stamp.fixed.state = mergeChain(stamp.fixed.state, Constructor.prototype);
```

Comment:
What is the objective you're trying to accomplish by using a state stamp here?

Would this still be necessary if we stopped filtering `.methods()` to prevent state from being stored on the prototype?

Should we even be doing that?